app/Enums/DatalinkAuthorities.php

```diff
@@ -13,6 +13,8 @@ enum DatalinkAuthorities: string
     case TTZO = 'TTZO';
     case CZQXD = 'CZQX';
     case CZQMD = 'CZQM';
+    case SYS = 'SYST';
+    case OCEN = "OCEN";
```

Comment:
nit: why is just this one in double quotes lol

Reply:
nothing to see here ([be184aa](https://github.com/vatsimnetwork/nattrak/pull/126/commits/be184aaad531c37908df7ddb6a7979f79adcf3be))